### PR TITLE
MGMT-4050 	E2E tests for disk ID

### PR DESCRIPTION
**Pull request by michaellevy101**
Support attach disk with a WWN(World Wide Name)
Having a WWN creates a disk by-id link (/dev/disk/by-id)

### CODE REVIEW

File: discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py [hunk 1]
```diff
@@ -220,12 +221,18 @@ def attach_test_disk(self, node_name, disk_size, bootable=False, persistent=Fals
         if persistent:
             attach_flags |= libvirt.VIR_DOMAIN_AFFECT_CONFIG
 
+        wwn = ""
+
+        if with_wwn:
```
Comment (omertuc) [inline discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py:226]: maybe reduce these 4 lines to a single ternary:
`wwn = f"<wwn>0x{secrets.token_hex(8)}</wwn>" if with_wwn else ""`

File: discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py [hunk 2]
```diff
@@ -220,12 +221,18 @@ def attach_test_disk(self, node_name, disk_size, bootable=False, persistent=Fals
...
+            token = secrets.token_hex(8)
+            wwn = f"""\n\t<wwn>0x{token}</wwn>"""
+
         node.attachDeviceFlags(f"""
             <disk type='file' device='disk'>
                 <alias name='{disk_alias}'/>
                 <driver name='qemu' type='qcow2'/>
                 <source file='{tmp_disk}'/>
-                <target dev='{target_dev}'/>
+                <target dev='{target_dev}'/>{wwn}
```
Comment (omertuc) [inline discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py:235]: You can put it on its own line, even if it's empty - XML shouldn't care

**Comment by michaellevy101** [comment]
@omertuc can you please approve it as well?

**Pull request merged**